MatrixSDK/Data/MXRoomAccountData.h

```diff
@@ -48,6 +48,9 @@
  */
 @property (nonatomic, readonly) MXVirtualRoomInfo *virtualRoomInfo;
 
+/// Any extra event received from the server.
+@property (nonatomic, readonly) NSDictionary <NSString*, NSDictionary<NSString*, id> * > *others;
```

Comment:
The problem if we expose only the space order, we'll have to force again the initial sync when another room account data event will be added and we want to avoid it.

I can call it `customEvents` if you want. No problem.

Reply:
The name is better, though if I am not missing something here, it still feels like we could have a read-only computed property `spaceOrderContent` in the interface of `MXRoomAccountData`, wheras internally it would keep the dictonary, and when requested, use the `kMXEventTypeStringSpaceOrderKey` to return the space ordering. The main difference is hiding the implementation details and a very loose contract where the `SpaceService` has to somehow know that `otherEvents` will contain `kMXEventTypeStringSpaceOrderKey`.